lwt-unix/jbuild

```diff
@@ -5,14 +5,14 @@
   (preprocess (pps (ppx_sexp_conv)))
   (wrapped false)
   (modules (resolver_lwt_unix conduit_lwt_unix conduit_lwt_server conduit_lwt_tls conduit_lwt_unix_ssl conduit_lwt_launchd))
-  (libraries (conduit-lwt tls uri.services ipaddr.unix logs 
+  (libraries (conduit-lwt tls.lwt uri.services ipaddr.unix logs
     (select conduit_lwt_launchd.ml from
      (launchd  -> conduit_lwt_launchd_real.ml)
      (!launchd -> conduit_lwt_launchd_dummy.ml))
     (select conduit_lwt_unix_ssl.ml from
-     (lwt.ssl  -> conduit_lwt_unix_ssl_real.ml)
-     (!lwt.ssl -> conduit_lwt_unix_ssl_dummy.ml))
+     (lwt_ssl  -> conduit_lwt_unix_ssl_real.ml)
+     (!lwt_ssl -> conduit_lwt_unix_ssl_dummy.ml))
```

Comment:
thanks! I believe this requires bumping the minimum lwt version to >=3.0.0 to account for the new package structure?